WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -105,6 +123,10 @@ private void updateSiteOrFinishActivity(Bundle savedInstanceState) {
         }
 
         if (mSite == null) {
+            mSite = mSiteStore.getSiteByLocalId(AppPrefs.getSelectedSite());
```

Comment:
Same comment as previously: `AppPrefs.getSelectedSite` should not be used outside the site picker and the "MySite" fragment.